Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,120 @@
+"""
+NCL_mask_5.py
+=============
+This script illustrates the following concepts:
+    - Using cartopy.feature options to display map information
+    - Paneling two plots on a page with 'subplots' command
+    - Drawing contours
+    - Explicitly setting the fill colors for contours
+    - Drawing contours over land only
+    - Using draw order resources to mask areas in a plot
+    - Adding a label bar
```

Comment:
```suggestion
    - Adding a color bar
```
Label bar is NCL terminology, so let's use equivalent Python terms instead